addons/point_of_sale/models/pos_order.py

```diff
@@ -1043,7 +1043,7 @@ def sync_from_ui(self, orders):
 
             if existing_draft_order:
                 order_ids.append(self._process_order(order, existing_draft_order))
-            else:
+            elif order.get('lines') or order.get('payment_ids'):
```

Comment:
The problem with this change is that if there are other changes in the order (like a state change) but the 'lines' is empty, then the order won't be processed. We can't go with this solution. We need to find another way.